avm/utilities/tools/Set-AVMModule.ps1

```diff
@@ -103,6 +109,42 @@ function Set-AVMModule {
         $relevantTemplatePaths = Join-Path $resolvedPath 'main.bicep'
     }
 
+    if (-not $SkipVersionCheck) {
+
+        # Get latest release from Azure/Bicep repository
+        # ----------------------------------------------
+        $latestReleaseUrl = 'https://api.github.com/repos/azure/bicep/releases/latest'
+
+        $latestReleaseObject = Invoke-RestMethod -Uri $latestReleaseUrl -Method 'GET'
+        $releaseTag = $latestReleaseObject.tag_name
+        $latestReleaseVersion = [version]($releaseTag -replace 'v', '')
+        $latestReleaseUrl = $latestReleaseObject.html_url
+
+        # Get latest installed Bicep CLI version
+        # --------------------------------------
+        $latestInstalledVersionOutput = bicep --version
+
+        if ($latestInstalledVersionOutput -match ' ([0-9]+\.[0-9]+\.[0-9]+) ') {
+            $latestInstalledVersion = [version]$matches[1]
+        }
+
+        # Compare the versions
+        # --------------------
+        if ($latestInstalledVersion -ne $latestReleaseVersion) {
+            Write-Warning """
+You're not using the latest available Bicep CLI version [$latestReleaseVersion] but [$latestInstalledVersion].
+You can find the latest release at: $latestReleaseUrl.
+
+On Windows, you can use chocolatey to update the Bicep CLI by running 'choco upgrade bicep'. For other OSs, please refer to the Bicep documentation (https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/install).
```

Comment:
Lets use `winget` here `winget install Microsot.Bicep` or `winget update Microsoft.Bicep`

Reply:
Implemented in PR #1139